tracing-forest/src/builder.rs

```diff
@@ -415,6 +418,31 @@ where
             extensions: self.extensions,
         }
     }
+
+    /// Adds an environment filter to this subscriber. This is based on the
+    /// [`tracing_subscriber`] [`EnvFilter`] and uses the same `RUST_LOG` syntax.
+    ///
+    /// If the `RUST_LOG` environment value is not found, this will default to `info`.
+    ///
+    /// # Examples
+    /// ```
+    /// # use tracing_forest::prelude::*;
+    /// tracing_forest::builder()
+    ///     .blocking_layer()
+    ///     .with_env_filter()
+    ///     .on_closure(|| {
+    ///         // do stuff here...
+    ///     })
+    /// ```
+    #[cfg(feature = "env-filter")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "env-filter")))]
```

Comment:
I don't think this needs a feature gate because it's not adding any dependencies. May as well have it built in.

Reply:
Yes, but it adds features to tracing-subscriber, which is why I put it with the gate :)